src/lib.rs

```diff
@@ -107,7 +107,7 @@ pub mod image;
 
 /// The reference importer.
 #[cfg(feature = "import_data_reference")]
-mod import;
+pub mod import;
```

Comment:
Could we avoid making this module public?

Reply:
Yeah, not sure why I had that in there